draft-ietf-quic-recovery.md

```diff
@@ -629,6 +629,14 @@ MAY use alternate strategies for determining the content of probe packets,
 including sending new or retransmitted data based on the application's
 priorities.
 
+If the handshake is complete, but not confirmed (see Section 4.1.1 and Section
```

Comment:
I think it might make sense to work this into the "Handshakes and New Paths" section above?

The way I'm now thinking of this is: It's the client's job to drive the handshake to confirmation, and here's what is necessary to ensure that occurs...  Does that make sense?

Reply:
Also, now that I read the existing text again, I'm wondering if the "Sending Probe Packets" section should be before "Handshakes and New Paths"?

Reply:
I will leave that change to you, but I have moved this up to the Handshakes and New Paths section.